earn/src/components/boost/ImportBoostWidget.tsx

```diff
@@ -217,14 +271,52 @@ export default function ImportBoostWidget(props: ImportBoostWidgetProps) {
     return { apr0, apr1 };
   }, [cardInfo, boostFactor, marketInfo]);
 
-  const { dailyInterest0, dailyInterest1 } = useMemo(() => {
-    if (!apr0 || !apr1) {
-      return { dailyInterest0: null, dailyInterest1: null };
+  useEffect(() => {
+    (async () => {
+      let theGraphClient = theGraphUniswapV3Client;
+      switch (activeChain.id) {
+        case arbitrum.id:
+          theGraphClient = theGraphUniswapV3ArbitrumClient;
+          break;
+        case optimism.id:
+          theGraphClient = theGraphUniswapV3OptimismClient;
+          break;
+        case goerli.id:
+          theGraphClient = theGraphUniswapV3GoerliClient;
+          break;
+        case mainnet.id:
+        default:
+          break;
+      }
+      const unixTwoDaysAgo = Math.floor(Date.now() / 1000) - 86400 * 2;
+      const initialQueryResponse = (await theGraphClient.query({
+        query: Uniswap24HourPoolDataQuery,
+        variables: {
+          poolAddress: cardInfo.uniswapPool.toLowerCase(),
+          date: unixTwoDaysAgo,
+        },
+        errorPolicy: 'ignore',
+      })) as ApolloQueryResult<UniswapV3GraphQL24HourPoolDataQueryResponse>;
+      if (initialQueryResponse.data.poolDayDatas) {
+        const poolDayData = initialQueryResponse.data.poolDayDatas[0];
+        setTwentyFourHourPoolData({
+          liquidity: GN.fromDecimalString(poolDayData.liquidity, 18).toDecimalBig(),
+          feesUSD: parseInt(poolDayData.feesUSD),
+        });
+      }
+    })();
+  }, [activeChain.id, cardInfo, setTwentyFourHourPoolData]);
+
+  const dailyInterestUSD = useMemo(() => {
+    if (!apr0 || !apr1 || !tokenQuotes) {
+      return 0;
```

Comment:
I feel like we should default to null so that we can display "unknown" or "loading" instead of lying and saying 0